src/GenerateFlexEndpointCommand.php

```diff
@@ -13,20 +13,25 @@
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 #[AsCommand(name: 'generate:flex-endpoint', description: 'Generates the json files required by Flex')]
 class GenerateFlexEndpointCommand extends Command
 {
+    public const PROVIDER_GITHUB = 'github';
+    public const PROVIDER_GITLAB = 'gitlab';
+
     protected function configure(): void
     {
         $this
-            ->addArgument('repository', InputArgument::REQUIRED, 'The name of the GitHub repository')
+            ->addArgument('repository', InputArgument::REQUIRED, 'The name of the repository')
             ->addArgument('source_branch', InputArgument::REQUIRED, 'The source branch of recipes')
             ->addArgument('flex_branch', InputArgument::REQUIRED, 'The branch of the target Flex endpoint')
             ->addArgument('output_directory', InputArgument::REQUIRED, 'The directory where generated files should be stored')
             ->addArgument('versions_json', InputArgument::OPTIONAL, 'The file where versions of Symfony are described')
             ->addOption('contrib')
+            ->addOption('provider', null, InputOption::VALUE_REQUIRED, 'Provider of the repository (github or gitlab)', self::PROVIDER_GITHUB)
```

Comment:
```suggestion
            ->addOption('provider', null, InputOption::VALUE_REQUIRED, 'Provider of the repository (github or gitlab)', self::PROVIDER_GITHUB, [self::PROVIDER_GITHUB, self::PROVIDER_GITLAB])
```

to provide completion